gaming-website/src/javaScript/games.json

```diff
@@ -0,0 +1,121 @@
+{
+  "payload": {
+    "gamesDetail": [
+      {
+        "name": "The Witcher 3: Wild Hunt",
+        "price": "40",
+        "img-cover": "the-witcher-3.png",
+        "img-highlight": "witcher-highlight.png",
+        "img-top-left": "witcher-gameplay-1.png",
```

Comment:
I would maybe keep the image highlight separate, and then all the rest of the images would be in an array.

Moving further away from how we will display the images.
On mobile devices will we have an image top left and an image top right?

Reply:
on mobile we will probably keep only the highlight image from the right that serves as a cover of the game